src/app/progress/PowerCaps.tsx

```diff
@@ -0,0 +1,34 @@
+import { powerLevelByKeyword } from 'app/search/d2-known-values';
+import clsx from 'clsx';
+import { DefaultTFuncReturn, t } from 'i18next';
```

Comment:
We use `import { t } from 'app/i18next-t';` which is our own import that corrects types. You shouldn't need `DefaultTFuncReturn`, you can use `string`.